snap/local/launcher.sh

```diff
@@ -1,2 +1,7 @@
 #!/bin/sh
-exec "$SNAP/opt/Mattermost/mattermost-desktop" --no-sandbox --disable-seccomp-filter-sandbox --disable-features=GlobalShortcutsPortal "$@"
+
+if [ -n "$WAYLAND_DISPLAY" ] && [ -z "$DISABLE_WAYLAND" ]; then
+  WAYLAND_OPTS="--enable-features=UseOzonePlatform --ozone-platform-hint=auto"
+fi
+
+exec "$SNAP/opt/Mattermost/mattermost-desktop" --no-sandbox --disable-seccomp-filter-sandbox --disable-features=GlobalShortcutsPortal $WAYLAND_OPTS "$@"
```

Comment:
I think I'd prefer to quote this variable, and include an empty default. Unlikely to ever cause an issue in this simple form, but my inner `shellcheck` is shouting at me ;-)

```suggestion
exec "$SNAP/opt/Mattermost/mattermost-desktop" --no-sandbox --disable-seccomp-filter-sandbox --disable-features=GlobalShortcutsPortal "${WAYLAND_OPTS:-}" "$@"
```

Reply:
You are correct, we should do that... but for some reason that doesn't fix the real issue I'm trying to fix.  On the Qualcomm based laptop, there are rendering issues when wayland isn't used in the electron apps right now, so it's really obvious when it does the wrong thing. Quoting that variable does pass the arguments properly, but electron isn't using wayland, if I don't quote it wayland is used as exected!

There's no visual difference in the cmd line passed in both cases... it's a head scratcher!
Without quoting:
```
/snap/mattermost-desktop/x4/opt/Mattermost/mattermost-desktop --no-sandbox --disable-seccomp-filter-sandbox --disable-features=GlobalShortcutsPortal --enable-features=UseOzonePlatform --ozone-platform-hint=auto
```

With quoting:
```
/snap/mattermost-desktop/x5/opt/Mattermost/mattermost-desktop --no-sandbox --disable-seccomp-filter-sandbox --disable-features=GlobalShortcutsPortal --enable-features=UseOzonePlatform --ozone-platform-hint=auto
```